src/routes/api/user.routes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import validation from '../../validation/user.validation';
+
+const router = express.Router();
+const {
+  checkFirstName, checkLastName, checkValidEmail, checkExistingEmail, checkPassword, validateResult
+} = validation;
+router.post('/signup', checkFirstName, checkLastName, checkValidEmail, checkExistingEmail, checkPassword, validateResult);
```

Comment:
```suggestion

router.post('/signup', checkFirstName, checkLastName, checkValidEmail, checkExistingEmail, checkPassword, validateResult);
```
Add space after constants